### PR TITLE
.NET Docker documentation

### PR DESCRIPTION
**Pull request by ldemidov**
Add section for docker installation.
Update supported technologies page.

@kaitlynmortimer the link in the supported techologies page should be update. 
Also you can decide whether to embed the links to Dockerfile and start.ps1 script, or embed them in the installation page.

### CODE REVIEW

File: content/installation/net/install/Supported-technology-net.md [hunk 1]
```diff
@@ -32,5 +32,17 @@ The .NET agent for Azure App Service supports analysis of web applications built
 | Processor Architecture     | Both 32-bit and 64-bit applications are supported. |
 | Web Application Frameworks | All of the frameworks supported by the Windows agent are analyzed on Azure. |
 
-
 > **Note:** Only applications using the full .NET Framework are currently supported. Contrast plans to add support for .NET Core applications in the future.
+
+## .NET for Docker
+
+The .NET agent can run in docker on a Windows based image that .NET Framework 4.5.1 and IIS.  We recommend basing your image on [microsoft/aspnet](https://hub.docker.com/_/microsoft-dotnet-framework-aspnet).  Please see [.NET Docker documentation](http://todo.com) for detailed installation instructions.
```
Comment (Silvenga) [inline content/installation/net/install/Supported-technology-net.md]: todo.com?

File: content/installation/net/install/Supported-technology-net.md [hunk 2]
```diff
@@ -32,5 +32,17 @@ The .NET agent for Azure App Service supports analysis of web applications built
...
+
+| Technology                 | Supported versions                       |
+| -------------------------- | ---------------------------------------- |
+| .NET Framework             | Your application must be set to use CLR4. Any application that can run on .NET 4.5.1 or above is supported.   |
```
Comment (Silvenga) [inline content/installation/net/install/Supported-technology-net.md]: If they are running in container mode, wouldn't they need net470 at a minimum?

File: content/installation/net/docker/Installation.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Then extract the nuget package and set the following environment variables on the application process.
+
+| Environment variable       | Value                       |
+| -------------------------- | --------------------------- |
+| COR_ENABLE_PROFILING       | 1                           |
+| COR_PROFILER               | {EFEB8EE0-6D39-4347-A5FE-4D0C88BC5BC1}   |
+| COR_PROFILER_PATH_32       | <Unzipped nuget package path>\content\contrastsecurity\ContrastProfiler-32.dll   |
+| COR_PROFILER_PATH_64       | <Unzipped nuget package path>\content\contrastsecurity\ContrastProfiler-64.dll   |
+| AGENT__DOTNET__CONTAINER   | true                        |
```
Comment (Silvenga) [inline content/installation/net/docker/Installation.md]: We need to monitor these double underscores when rendering the markdown.

File: content/installation/net/install/Supported-technology-net.md [hunk 4]
```diff
@@ -32,5 +32,17 @@ The .NET agent for Azure App Service supports analysis of web applications built
...
+| CLR                        | CLR4                       |
+| Processor Architecture     | Both 32-bit and 64-bit applications are supported. |
+| Web Application Frameworks | All of the frameworks supported by the Windows agent are analyzed on Azure. |
```
Comment (Silvenga) [inline content/installation/net/install/Supported-technology-net.md]: > Supported Web Application Frameworks: All of the frameworks supported by the Windows agent are analyzed on Azure.
This is a little confusing, but I'm not sure how to re-word.

File: content/installation/net/install/Supported-technology-net.md (hunk 1 above)
Comment (ldemidov) [reply content/installation/net/install/Supported-technology-net.md]: fixed but kaitlyn would need to set the final link

Comment (ldemidov) [reply content/installation/net/install/Supported-technology-net.md]: Put the examples in a separate repo and linked to them.

File: content/installation/net/docker/Installation-docker.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+<!--
+title: "Installing Contrast .NET Framework agent in Docker"
+description: "Guide to installing .NET agent on Docker for Windows containers using the NuGet package"
+tags: "installation configuration .net docker nuget visual studio"
+-->
+
+The following instructions show you how install and configure the Contrast .NET agent in Docker for Windows using the NuGet package. Examples of the finished code exist in a [GitHub repository](https://github.com/Contrast-Security-OSS/contrast-dotnet-examples).
```
Comment (kaitlynmortimer) [inline content/installation/net/docker/Installation-docker.md]: @ldemidov Did I describe this accurately?

Comment (ldemidov) [reply content/installation/net/docker/Installation-docker.md]: yep

**Pull request merged**